### ISSUE TITLE
Better disabled button

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#888]
When disabling buttons the left and top highlight are still shown as they were active which looks quite odd. The highlight brightness should be adjusted - or just use the buttons normal texture which would result in the following: 
![image](https://user-images.githubusercontent.com/1276342/47839926-d314af00-ddb4-11e8-8a75-39d5d0b45a75.png)
(left current, right new)

**Comment by Spikeone** [comment]
Although I agree that both versions are not perfect, I think the one on the right is slightly better. The one on the left doesn't look like a 3D button either as the bright border just not fits, at least for me. Maybe disabled buttons should just be pressed and have no highlight onMouseover - that way it looks like the button is pressed... so you can't do it anymore?

**Comment by Spikeone** [comment]
Okay, so as I agreed, that the solution is not perfect, I checked the original.
First of all they do use only one color for the letters (117,125,134). Second they use a different, less dark color for the disabled textures. They do look more greyed out at all and make the text have less contrast. I then tried to find what they did (took 4 original pixels and compared their RGB to the greyed texture).
`orig ->     disabled -> diff
35,71,27    32,36,32    -3 ,-35, +5
95,91,67    60,60,69    -35,-32, -2
7 ,47,35    0 ,24,32    -7 ,-23, -3
0 ,27,35    0 ,24,32     0 , -3, -3`
Not much help. I think this is done using palettes to map the colors. 

But here the example: left = current, middle = grey letters, right = grey letters and texture
![image](https://user-images.githubusercontent.com/1276342/47934603-fbe19500-ded7-11e8-9e2c-251b4a803d90.png)

In my opinion the outcome is worth the work as it looks really good and we need such buttons for campaigns as well.
